bootstrap/src/main/scala/org/polyvariant/Main.scala

```diff
@@ -20,35 +21,51 @@ object Main extends IOApp {
       Logger[F].info(s"ID: ${mr.mergeRequestIid} by: ${mr.authorUsername}")
     }.void
 
-  private def readConsent[F[_]: Console: Applicative]: F[Boolean] =
-    Console[F].readLine.map(_.toLowerCase == "y")
+  private def readConsent[F[_]: Console: MonadThrow]: F[Unit] =
+    MonadThrow[F]
+      .ifM(Console[F].readLine.map(_.toLowerCase == "y"))(
```

Comment:
```suggestion
      .ifM(Console[F].readLine.map(_.trim.toLowerCase == "y"))(
```

just for good measure